spec/models/miq_widget_spec.rb

```diff
@@ -130,18 +130,18 @@
       end
 
       it "ignores the legacy format admin|db_name" do
-        expect { FactoryBot.create(:miq_widget_set, :set_data_with_one_widget, :set_data_with_one_widget, :name => "#{@user1.userid}|Home", :owner => @user1.current_group) }.to raise_error(ActiveRecord::RecordInvalid)
+        expect { FactoryBot.create(:miq_widget_set, :name => "#{@user1.userid}|Home", :owner => @user1.current_group) }.to raise_error(ActiveRecord::RecordInvalid)
       end
 
       context 'with subscribers' do
         before do
-          ws = FactoryBot.create(:miq_widget_set, :set_data_with_one_widget, :name => "Home", :userid => @user1.userid, :group_id => @group1.id)
+          ws = FactoryBot.create(:miq_widget_set, :name => "Home", :userid => @user1.userid, :group => @group1)
           @widget_report_vendor_and_guest_os.make_memberof(ws)
         end
 
         it "returns non-empty array when widget has subscribers" do
           user_temp = add_user(@group1)
-          ws_temp   = add_dashboard_for_user("Home", user_temp.userid, @group1.id)
+          ws_temp   = add_dashboard_for_user("Home", user_temp.userid, @group1)
```

Comment:
I can't comment on the right line, but the expectations on this spec are now wrong. at a min,
`expect(result.size).to eq(1) # should be 2`
the other expectation's also going to be wrong but I didn't look into why, it's going to get the user created in the default widget instead of the temp